.github/workflows/create-release-draft.yml

```diff
@@ -145,17 +138,17 @@ jobs:
 
       - name: Push docker image
         run: |
-          docker push litentry/litentry-parachain:$PARACHAIN_DOCKER_TAG
+          docker push litentry/litentry-parachain:${{ env.RELEASE_TAG }}
```

Comment:
I would suggest pushing at the end of the workflow - to make sure we publish only once all builds/tests/checks pass.

Otherwise some images might be published some not.